### PR TITLE
Add autofix for at-if/else-closing-brace-newline-after

### PR DESCRIPTION
**Pull request by OriR**

### CODE REVIEW

File: src/rules/at-else-closing-brace-newline-after/__tests__/index.js [hunk 1]
```diff
@@ -80,6 +89,11 @@ testRule(rule, {
 
       } @else if { }
       @else { }
+    }`,
+      fixed: `a {
+      @if ($x == 1) {
+
+      } @else if { }@else { }
```
Comment (OriR) [inline src/rules/at-else-closing-brace-newline-after/__tests__/index.js]: Not sure this fix makes sense 😕

File: src/rules/at-else-closing-brace-newline-after/__tests__/index.js [hunk 2]
```diff
@@ -92,6 +106,11 @@ testRule(rule, {
       } @else { }
 
       @else { }
+    }`,
+      fixed: `a {
+      @if ($x == 1) {
+
+      } @else { }@else { }
```
Comment (OriR) [inline src/rules/at-else-closing-brace-newline-after/__tests__/index.js]: Same here...

File: src/rules/at-else-closing-brace-newline-after/__tests__/index.js (hunk 1 above)
Comment (kristerkari) [reply src/rules/at-else-closing-brace-newline-after/__tests__/index.js]: In my opinion `} @else if { } @else { }` would look better :)

Comment (OriR) [reply src/rules/at-else-closing-brace-newline-after/__tests__/index.js]: Makes sense, that's the same approach as in #221 👍

**Comment by kristerkari** [comment]
Looks like you did some indentation changes in a0c2011 that broke the tests.

**Comment by OriR** [comment]
@kristerkari Fixed it 👍 

I think we're good to go on all auto-fix PRs, right?

**Comment by kristerkari** [comment]
Hmm... I'm thinking that maybe this rule should also have `disableFix` as it adds a newline without indentation. What do you think @OriR ?

**Comment by OriR** [comment]
Yeah, on second look it looks odd.
Will add it now.

**Pull request merged**